Tasks/XamariniOS/xamarinios.ts

```diff
@@ -5,29 +5,44 @@ import msbuildhelpers = require('msbuildhelpers/msbuildhelpers');
 
 import { ToolRunner } from 'vsts-task-lib/toolrunner';
 
-async function run() {
+function expandSolutionWildcardPatterns(solutionPattern: string): string {
+    const matchedSolutionFiles = tl.findMatch(null, solutionPattern, { followSymbolicLinks: false, followSpecifiedSymbolicLink: false });
```

Comment:
As Bryan pointed out, we always want to search from the root of the repository.  This is consistent with how the task works right now and will avoid breaking existing definitions.

Reply:
Thanks. I did some testing with the Xcode task and found because `xcWorkspacePath` is a filePath argument, it always comes into our task as a rooted path (*.xcodeproj/project.xcworkspace => /Users/vsts/agent/2.126.0/work/1/s/*.xcodeproj/project.xcworkspace). Therefore the first argument to findMatch is ignored.

Sorry for the noise.